src/main/java/org/demo/conf/cxbox/extension/drilldown/CustomFilterBuilder.java

```diff
@@ -0,0 +1,53 @@
+package org.demo.conf.cxbox.extension.drilldown;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import javax.annotation.Nullable;
+import lombok.NonNull;
+import org.cxbox.api.data.dto.DataResponseDTO;
+import org.cxbox.constgen.DtoField;
+import org.cxbox.core.controller.param.SearchOperation;
+import org.cxbox.core.service.drilldown.filter.FB;
+import org.cxbox.core.service.drilldown.filter.PlatformUrlParametersFilterService;
+
+/**
+ * Example use custom filter builder
+ * @param <D> DTO
+ */
+public class CustomFilterBuilder<D extends DataResponseDTO> extends
```

Comment:
move this doc java doc and code-samples. demo should stay as compact as possible.